options.html

```diff
@@ -39,7 +39,7 @@
   <div id="leftnav">
     <div id="articleSort">Sorted by Most Recent</div>
     <div id="article-scrollContainer"></div>
-    <span id="capture-helper">&#8984;+Shift+S to save a quote on any site</span>
+    <span id="capture-helper">Ctrl+Shift+S / &#8984;+Shift+S to save a quote on any site</span>
```

Comment:
When I went looking in the extension for what key to use, it just says "cmd" here, which isn't too useful for me because I'm not on a Mac.
This should help people out more